yolov6/logger/wandb_inference_logger.py

```diff
@@ -0,0 +1,56 @@
+import cv2
+import wandb
+
+
+class WandbInferenceLogger:
+    def __init__(self) -> None:
+        self.label_dictionary = {}
+        self.table = wandb.Table(
+            columns=[
+                "Image-File",
+                "Predictions",
+                "Number-of-Objects",
+                "Prediction-Confidence",
+            ]
+        )
+
+    def set_label_dictionary(self, label_dictionary):
```

Comment:
nice opportunity to use python properties.

change label_dictionay in your `__init__ `
```
self._label_dictionary =  {}
```

then

```
@property
def label_dictionary(self):
    return self._label_dictionary

@propery. setter
def label_dictionary(self, new_dict):
    self._label_dictionary = new_dict
```

then in your code reference:
```
self.inference_logger.label_dictionary = 
                {idx: self.class_names[idx] for idx in range(len(self.class_names))}
```


[Why this](https://www.programiz.com/python-programming/property)

Reply:
Thanks for this suggestion 😁